app/models/generates_notifications.rb

```diff
@@ -0,0 +1,13 @@
+module GeneratesNotifications
+    extend self
+
+    def generate_notifications_for(users, title:)
+        users.each do |user|
```

Comment:
Indentation is off. You may also wanna use [`#find_each`](http://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each) for the bulk of it.